components/avatar/avatar.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../component.css">
+    <link rel="stylesheet" href="../main.css">
+    <link rel="stylesheet" href="../reset.css">
+    <link rel="stylesheet" href="avatar.css">
+    <link rel="icon" href="../../assets/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Avatar | bloom UI</title>
+</head>
+<body>
+    <nav class="navigation-bar">
+        <div class="logo">
+            <a href="../index.html">
+            <img  class="logo-img" src="../assets/logo1.svg" alt="logo">
+            <p class="logo-text">bloom</p><span class="text">UI</span>
+            </a>
+        </div>
+        <div class="hamburger-icon">
+            <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
+        </div>
+    </nav>
+    <div class="conatiner">
+        <div class="sidebar-wrapper slide-off">
+            <div class="sidebar">
+                <div class="sidebar-header">
+                    <div class="logo">
+                        <a href="../index.html">
+                        <img class="logo-img" src="../assets/logo1.svg" alt="logo">
+                        <p class="logo-text">bloom</p><span class="text">UI</span>
+                        </a>
+                    </div>
+                </div>
+                <div class="introduction">
+                    <a href="main.html">Introduction</a>
+                </div>
+                <div class="components">Components</div>
+                <div class="sidebar-main">
+                    <a href="../alert/alert.html">Alert</a>
+                    <a href="../avatar/avatar.html">Avatar</a>
+                    <a href="../badge/badge.html">Badge</a>
+                    <a href="">Button</a>
+                    <a href="">Card</a>
+                    <a href="">Image</a>
+                    <a href="">Input</a>
+                    <a href="">Lists</a>
+                    <a href="">Modal</a>
+                    <a href="">Navigation</a>
+                    <a href="">Rating</a>
+                    <a href="">Slider</a>
+                    <a href="">Toast</a>
+                    
+                </div>
+            </div>
+        </div>
+        <div class="section">
```

Comment:
I suggest not to use keywords for class names. Example, 'section' is a tag in HTML and using the same word for the class name might create confusion.